frontend/src/Question.jsx

```diff
@@ -21,7 +21,9 @@ function Question({ question, questionId, onDelete }) {
   const [description, setDescription] = React.useState("");
 
   const loadDesciption = async () => {
-    const response = await axios.get("http://localhost:8080/question/" + question.id);
+    const response = await axios.get(
+      "http://localhost:8080/question/" + question.id
```

Comment:
base url (before "/question") should be abstracted in a future pr, maybe in a .env file

Reply:
okay will take note, thanks!